lib/include/jxl/decode.h

```diff
@@ -1199,8 +1199,8 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderSetDecompressBoxes(JxlDecoder* dec,
  *
  * @param dec decoder object
  * @param type buffer to copy the type into
- * @param decompressed which box type to get: JXL_TRUE to get the raw box type,
- * which can be "brob", JXL_FALSE, get the underlying box type.
+ * @param decompressed which box type to get: JXL_FALSE to get the raw box type,
```

Comment:
Can you move this to a separate PR? Seems fairly unrelated.

Reply:
Right, it's 'related' because I found this documentation bug while working on jxlinfo, but it's indeed better in a separate PR. See https://github.com/libjxl/libjxl/pull/1363